tests/libfixmath/main.c

```diff
@@ -184,7 +191,10 @@ static void unary_ops(void)
 
 int main(void)
 {
+    /* Delay output to prevent flooding of buffer */
+    dumb_delay(1000000);
```

Comment:
Please use a macro here (e.g `#define UART_DELAY 1000000`)

Reply:
Something like that.